tests/integration_tests/nn/fixed_point/batchnorm_linear/layer_design_test.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+
+import elasticai.creator.nn.fixed_point as nn_creator
+from elasticai.creator.nn import Sequential
+from elasticai.creator.nn.fixed_point.math_operations import FixedPointConfig
+from tests.integration_tests.nn.fixed_point.sequential_routine import (
+    routine_testing_sequential_module,
```

Comment:
I think this function could use a rename. Imo routine just being a different name for function this is just saying it has smth to do with testing the sequential module. I assume that is calling the simulator? Maybe call it `run_simulation_for_sequential`?